api/v1alpha1/experiment_types.go

```diff
@@ -36,9 +36,11 @@ type Parameter struct {
 	// The name of the parameter
 	Name string `json:"name"`
 	// The inclusive minimum value of the parameter
-	Min int64 `json:"min,omitempty"`
+	Min int32 `json:"min,omitempty"`
 	// The inclusive maximum value of the parameter
-	Max int64 `json:"max,omitempty"`
+	Max int32 `json:"max,omitempty"`
+	// Internal use only
+	Values []string `json:"values,omitempty"`
```

Comment:
Just going to quote the Kubebuilder book:
> In Kubernetes, all versions must be safely round-tripable through each other. This means that if we convert from version 1 to version 2, and then back to version 1, we must not lose information. Thus, any change we make to our API must be compatible with whatever we supported in v1, and also need to make sure anything we add in v2 is supported in v1. In some cases, this means we need to add new fields to v1, but in our case, we won’t have to, since we’re not adding new functionality.